### ISSUE TITLE
dropout & drop_connect 

### ISSUE THREAD

**Issue by M-junjun** [lukemelas/EfficientNet-PyTorch#323]

**Comment by M-junjun** [comment]
when use pre-trained model, can I change dropout rate and drop_connect rate?

**Comment by M-junjun** [comment]
this issue is resolved.
I didn't confirm enough. I'm sorry.
